### PR TITLE
Table

### PR DESCRIPTION
**Pull request by SamDavies**
Screenshot 2020-01-27 at 23 38 24

### CODE REVIEW

File: src/components/Checkbox/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const StyledCustomCheckbox = styled.span`
     border: 2px solid ${colors.black};
     height: 14px;
     width: 14px;
-    transition: 150ms ease-in;
+    transition: 30ms ease-in;
```
Comment (lesbaa) [inline src/components/Checkbox/index.js]: a 30ms transition, is that even noticeable?

File: src/components/TableCell/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    ${props => props.flex && `
+        flex: ${props.flex}
+    `}
+    ${props => props.width && `
+        width: ${props.width}
+    `}
+    padding-top: 12px;
+    padding-left: 10px;
+    padding-right: 10px;
+    padding-bottom: 8px;
```
Comment (lesbaa) [inline src/components/TableCell/index.js]: `padding: 12px 10px 8px 10px`, no?

File: src/components/TableHeader/index.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+import { colors } from '../../styles'
+
+export const TableHeader = styled.div`
+    display: flex;
+    flex-direction: row;
```
Comment (lesbaa) [inline src/components/TableHeader/index.js]: `flex-direction: row` is the default, so unless this is an override you probably don't need this.

File: src/components/TableRow/index.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+import { colors } from '../../styles'
+
+export const TableRow = styled.div`
+    display: flex;
+    flex-direction: row;
```
Comment (lesbaa) [inline src/components/TableRow/index.js]: same with this guy

File: src/components/Checkbox/index.js (hunk 1 above)
Comment (SamDavies) [reply src/components/Checkbox/index.js]: oh yeah 30ms is really small, it seemed really slow before

File: src/components/TableCell/index.js (hunk 2 above)
Comment (SamDavies) [reply src/components/TableCell/index.js]: I actually prefer writing it more verbosely, then you don't have to remember the order

File: src/components/TableHeader/index.js (hunk 3 above)
Comment (SamDavies) [reply src/components/TableHeader/index.js]: yes true

**Pull request merged**